.github/workflows/UnitTesting.yaml

```
@@ -16,13 +16,14 @@ jobs:
       py39: 3.9
       py310: '3.10'
       py311: '3.11'
+      py311: '3.12'
```

Should this be `py312`? 
```suggestion
      py312: '3.12'
```